WordPress/Classes/ViewRelated/Jetpack/JetpackInstallPromptSettings.swift

```diff
@@ -31,7 +31,7 @@ public final class JetpackInstallPromptSettings {
             return false
         }
 
-        return !jetpack.isInstalled && blog.isAdmin && !promptWasDismissed(for: blog)
+        return !jetpack.isConnected && blog.isAdmin && !promptWasDismissed(for: blog)
```

Comment:
To clarify my above comment, the change would look something like this:
```suggestion
        let isJetpackConnectedToWPAccount = jetpack.isInstalled && blog.account != nil
        return !isJetpackConnectedToWPAccount && blog.isAdmin && !promptWasDismissed(for: blog)
```
I'm aware that this is the "Jetpack **_Install_** Prompt", so having it also prompt users to connect Jetpack to their WP.com accounts could be scope creep. 
That said, a user who doesn't connect their WP.com account will need to log in via their site credentials the next time they want to log in to their site via the app.

Reply:
Thanks for the review and suggestions!

> Should we show a prompt in this scenario (e.g. a "Jetpack plugin WP.com account connection" prompt), to encourage the user to finish the connection and unlock all of these Jetpack features?

Yes, we definitely should. It will be implemented in the next phase. 

> I'm aware that this is the "Jetpack Install Prompt", so having it also prompt users to connect Jetpack to their WP.com accounts could be scope creep.

Yes, that is an increase in scope.  To implement such logic properly, we would need to:
- Add  presentation logic to show WP.com login screen instead of Install screen,  if Jetpack is connected but the WP.com account doesn't exist
- If we want to have friendly UX, replace "Install prompt" with some new "Login to WP.com" prompt, if Jetpack is connected but the WP.com account doesn't exist

If we change the code as in the suggestion:
- "Install prompt" is still shown in the case if Jetpack is connected and the WP.com account doesn't exist
- When we click "Install" we are taken to the installation flow
- Installation flow fails, since Jetpack is already installed and connected

<img src="https://user-images.githubusercontent.com/4062343/186841692-a271cbc2-a081-406f-8605-4520ea23247a.png" alt="drawing" width="200"/>

Therefore, such change wouldn't be enough for now.